src/client.rs

```diff
@@ -173,3 +163,12 @@ impl<R: RetryPolicy<E = VssError>> VssClient<R> {
 		}
 	}
 }
+
+fn build_client() -> Client {
+	Client::builder()
+		.timeout(DEFAULT_TIMEOUT)
+		.connect_timeout(DEFAULT_TIMEOUT)
+		.read_timeout(DEFAULT_TIMEOUT)
+		.build()
+		.unwrap()
+}
```

Comment:
Could probably inline this in `from_client` (does that need to be `pub`?) and pass `header_provider` from each `new` method.